src/Evolution/DiscontinuousGalerkin/Initialization/Mortars.hpp

```diff
@@ -0,0 +1,134 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <boost/functional/hash.hpp>
+#include <cstddef>
+#include <tuple>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "Domain/Structure/Direction.hpp"
+#include "Domain/Structure/Element.hpp"
+#include "Domain/Structure/Neighbors.hpp"
+#include "Domain/Structure/OrientationMap.hpp"
+#include "Domain/Tags.hpp"
+#include "Evolution/DiscontinuousGalerkin/MortarData.hpp"
+#include "Evolution/DiscontinuousGalerkin/MortarTags.hpp"
+#include "NumericalAlgorithms/Spectral/Mesh.hpp"
+#include "Time/Tags.hpp"
+#include "Time/TimeStepId.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace Parallel {
+template <typename Metavariables>
+class GlobalCache;
+}  // namespace Parallel
+namespace tuples {
+template <class... Tags>
+class TaggedTuple;
+}  // namespace tuples
+/// \endcond
+
+namespace evolution::dg::Initialization {
+/*!
+ * \brief Initialize mortars between elements for exchanging boundary correction
+ * terms.
+ *
+ * If the template parameter `AddFluxBoundaryConditionMortars`
+ * is set to `false` then the mortar data for flux boundary conditions are not
+ * initialized and other boundary conditions can be applied. In this case, the
+ * `Tags::Mortars` items have no entries for external boundary directions.
```

Comment:
I don't know what `Tags::Mortars` refers to as it is a PrefixTag that does not appear to be used